protocol-configuration/mainnetAddresses.ts

```diff
@@ -855,6 +885,19 @@ export const MainnetContractsConfig = {
     category: AddressCategory.External
   },
 
+  aaveCompaniesMultisig: {
+    artifactName: 'unknown',
+    address: '0xe8d4D93D9728bD673b0197673a230F62255C7846',
+    category: AddressCategory.External
+  },
+
+  // TODO: Update to correct multisig, and not Aave companies when ready
+  aaveCompaniesDaiEscrowMultisig: {
```

Comment:
Note this TODO. Important we update to be the correct escrow multisig - currently I've put in the standard Aave companies multisig so I can test the proposal.

Reply:
done